### PR TITLE
Add a license header

### PR DESCRIPTION
**Pull request by khal0988**
Associated Issue: #2925

Here's the Pull Request Doc
https://devtools-html.github.io/debugger.html/CONTRIBUTING.html#pull-requests

### Summary of Changes

* added license header to .js files in src
* added license header to .js files in src/actions
* added license header to .js files in src/client
* added license header to files in src/components
* added license header to files in src/reducers

Here's the Debugger's Testing doc
https://docs.google.com/document/d/1oBMRxV8A2ag2t22YsQOxTdEv0mXKzIg0tggJjRkU1S0/edit#.
Feel free to improve it!

### CODE REVIEW

File: src/actions/utils/middleware/history.js [hunk 1]
```diff
@@ -1,9 +1,8 @@
-// @flow
-
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-/* global window */
+ * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
```
Comment (jasonLaster) [inline src/actions/utils/middleware/history.js]: we still need `- /* global window */`

File: src/actions/utils/middleware/wait-service.js [hunk 2]
```diff
@@ -1,9 +1,8 @@
-// @flow
-
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-/* global window */
+ * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
```
Comment (jasonLaster) [inline src/actions/utils/middleware/wait-service.js]: here too

File: src/actions/utils/middleware/history.js (hunk 1 above)
Comment (khal0988) [reply src/actions/utils/middleware/history.js]: whoops! I accidentally deleted it, fixed this

File: src/actions/utils/middleware/wait-service.js (hunk 2 above)
Comment (khal0988) [reply src/actions/utils/middleware/wait-service.js]: here too

File: src/components/WelcomeBox.js [hunk 3]
```diff
@@ -1,5 +1,9 @@
 // @flow
 import React, { Component } from "react";
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
```
Comment (jasonLaster) [inline src/components/WelcomeBox.js]: this should be on top...

**Pull request merged**